modules/administration/pages/disconnected-setup.adoc

```diff
@@ -151,75 +155,50 @@ SMT:: https://documentation.suse.com/sles/12-SP5/single-html/SLES-smt/index.html
 
 
 
-== Synchronize a Disconnected Server
 
-When you have removable media loaded with your {scc} data, you can use it to synchronize your disconnected server.
+[[disconnected-setup-server]]
+== Deploy and Synchronize a Disconnected Server
 
+It is recommended to deploy {productname} Server as a Virtual Machine (VM).
+The VM image comes with all needed tools included.
+For a step-by-step deployment procedure, see xref:installation-and-upgrade:container-deployment/suma/server-deployment-vm-suma.adoc[].
```

Comment:
I would just replace this by link to https://documentation.suse.com/suma/5.0/en/suse-manager/installation-and-upgrade/container-deployment/suma/server-air-gapped-deployment-suma.html

Because this section is about preparing SUMA/Uyuni for disconnected environments and not about actual installation of SUMA server.

Reply:
i.e. `For air-gapped installation of {productname} Server see https://documentation.suse.com/suma/5.0/en/suse-manager/installation-and-upgrade/container-deployment/suma/server-air-gapped-deployment-suma.html`

Important bit from this page is that `mgradm install podman` should have `--mirrorPath` option passed to it.

Reply:
Thanks, I tried to change it accordingly.